src/logion-chain/LogionLOC.tsx

```diff
@@ -0,0 +1,25 @@
+import { ApiPromise } from '@polkadot/api';
+import { ExtrinsicSubmissionParameters, signAndSend, Unsubscriber } from './Signature';
+import { UUID } from './UUID';
+
+export interface LOCCreationResult extends ExtrinsicSubmissionParameters {
```

Comment:
Name `LOCCreationResult` is a bit misleading IMHO. `LOCCreationParameters` sounds better.
```suggestion
export interface LOCCreationParameters extends ExtrinsicSubmissionParameters {
```

Reply:
Indeed.